### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by mbax**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+labels: ''
+assignees: mbax
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Stacktrace**
+Paste the stacktrace you get here.
```
Comment (kashike) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Ideally using a paste service, such as https://paste.gg.

**Comment by Zarthus** [comment]
A paste service? On github? Why
?

**Comment by jkcclemens** [comment]
```
<@jkcclemens>	I guess you could put it in the first section, but it's weird seeing a "expected behaviour" without an "actual behaviour"
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Zarthus) [reply .github/ISSUE_TEMPLATE/bug_report.md]:
```diff
diff --git a/a b/b
index 124e49f..0de14c2 100644
--- a/a
+++ b/b
@@ -1,2 +1,5 @@
 **Stacktrace**
+
+```java
 Paste the stacktrace you get here.
+```
```

apperantly you need write access to use [comment suggestions](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/), silly

Comment (mbax) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I'm not (yet) convinced of the need to keep it off-github.

Comment (bendem) [reply .github/ISSUE_TEMPLATE/bug_report.md]: especially since github supports details tags that can be used in templates to hide long snippets.

**Pull request merged**